cypress/integration/governanceReviewTeam.spec.js

```diff
@@ -0,0 +1,53 @@
+describe('Governance Review Team', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/api/v1/system_intakes?status=open').as('getOpenIntakes');
+    cy.localLogin({ name: 'GRTB', role: 'EASI_D_GOVTEAM' });
+    cy.visit('/');
+  });
+
+  it('can add GRT/GRB dates', () => {
+    cy.wait('@getOpenIntakes');
+
+    // Selecting name based on pre-seeded data
+    // A Completed Intake Form - af7a3924-3ff7-48ec-8a54-b8b4bc95610b
+    cy.get('a').contains('A Completed Intake Form').click();
```

Comment:
This intake is pre-seeded. The `name` and `id` are set. This would break if the name in the seed data changed.

Maybe there's a better way to target? I'd imagine this pre-seeded intake's name would be very unlikely to change.